content/en/docs/distributions/gke/deploy/overview.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Overview"
+description = "Full fledged Kubeflow deployment on Google Cloud"
+weight = 1
++++
+
+This guide describes how to deploy Kubeflow and a series of Kubeflow components on GKE (Google Kubernetes Engine). 
+If you want to use Kubeflow Pipelines only, redirect to [Kubeflow Pipelines](/docs/components/pipelines/) for installation
+guide. Refer to [Installation Options for Kubeflow Pipelines](/docs/components/pipelines/installation/overview/)
+for choosing an installation option.
+
+### Features
+
+Once you finish deployment, you will be able to:
+
+1. manage a running Kubernetes cluster with multiple Kubeflow components installed.
+1. get a [Cloud Endpoint](https://cloud.google.com/endpoints/docs) which is accessible via [IAP (Identity-aware Proxy)](https://cloud.google.com/iap).
+1. enable [Multi-user feature](/docs/components/multi-tenancy/) for resource and access isolation.
+1. take advantage of GKE's
+  [Cluster Autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler) 
+  to automatically resize the number of nodes in a node pool.
+1. choose GPUs and [Cloud TPU](https://cloud.google.com/tpu/) to accelerate your workload.
+1. use [Stackdriver](https://cloud.google.com/logging/docs/) to help debugging and troubleshooting.
```

Comment:
nit: looks like stackdriver is renamed to Cloud Logging

Reply:
Sounds good! Updated.